crossaccountathena.cf.yaml

```diff
@@ -7,19 +7,58 @@ Parameters:
     Description: AWS account ID for Glue Data Catalog
     AllowedPattern: \d{12}
     ConstraintDescription: Must be an AWS account ID
+  GlueDataCatalogRegion:
+    Type: String
+    Description: (Optional) AWS region ID for Glue Catalog (Eg. us-east-1). Defaults to current region if not specified.
+  AthenaCatalogName:
+    Type: String
+    Description: Athena Catalog name for custom data source
+    AllowedPattern: ^[a-z0-9_@-]*$
+    ConstraintDescription: Must be lowercase alphabets, digits, underscore, or hyphen
+  LambdaCodeS3Bucket:
+    Type: String
+    Description: S3 bucket containing function code zip file
+  LambdaCodeS3Key:
+    Type: String
+    Description: S3 path to function zip file. Eg. prefix/functionv2.zip
+  S3SpillLocation:
+    Type: String
+    Description: (Optional) S3 path to spill Lambda response so that it doesn't exceed Lambda's limit. Eg. s3://<bucket>/<spill-prefx>/
```

Comment:
I also thought about it. I chose it be optional because not all users have large number of partitions in their tables to require spilling. Anyway, the code checks whether environment variable is set or not, so we can remove the "(Optional)" wording.